dataeng/jobs/analytics/PipelineAcceptanceTestMaster.groovy

```diff
@@ -0,0 +1,70 @@
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+
+
+class PipelineAcceptanceTestMaster {
+
+    public static def job = { dslFactory, allVars ->
```

Comment:
I believe we should require that all new jenkins jobs include a useful `description()`

Reply:
You are right. Let me fix that